src/langcheck/metrics/zh/reference_free_text_quality.py

```diff
@@ -217,10 +221,96 @@ def _toxicity_local(generated_outputs: List[str]) -> List[float]:
     )
     # labels except Normal are all risky, toxicity_score = 1-score['Normal']
     toxicity_scores = []
-    for item_predict_proba in _predict_results:
+    for item_predict_proba in _predict_results:  # type: ignore[reportOptionalIterable]  # NOQA: E501
         for label_proba in item_predict_proba:  # type: ignore[reportGeneralTypeIssues]  # NOQA: E501
             # yapf: disable
             if label_proba['label'] == _model_id2label[0]:  # type: ignore[reportGeneralTypeIssues]  # NOQA: E501
                 toxicity_scores.append(1 - label_proba['score'])  # type: ignore[reportGeneralTypeIssues]  # NOQA: E501
             # yapf: enable
     return toxicity_scores  # type: ignore[reportGeneralTypeIssues]
+
+
+def xuyaochen_report_readability(
+        generated_outputs: List[str]) -> MetricValue[float]:
```

Comment:
Let's add `prompts: Optional[List[str] | str] = None` as a parameter here, and then add the prompts to the metric value output. (`flesch_reading_ease` and `tateishi_ono_yamada_reading_ease` have this too)